prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -108,6 +109,12 @@ modules_enabled = {
 		"s2sout_override";
 		"s2s_whitelist";
 		{{ end -}}
+
+		-- metrics collection functionality
+		{{ if $PROSODY_ENABLE_METRICS }}
+		"http_openmetrics";
+		{{ end }}
```

Comment:
```suggestion
		{{ end -}}
```
This will remove some whitespace introduced by the templating.